example/november_2017_evaluation/false_positives/cheapskate_get_image.py

```diff
@@ -0,0 +1,13 @@
+@app.route('/image', methods = ['GET'])
+def get_image():
+  url = request.args.get('url', '')
+  if not url:
+    abort(400)
+
+  if is_image(url):
```

Comment:
This one isn't a false-positive, but it easily could have been, in order to do this well we would need to be able to solve predicates, via like [Z3str3](https://sites.google.com/site/z3strsolver/) or something. I feel these are acceptable for the foreseeable future.